static/app/views/projectInstall/createProject.tsx

```diff
@@ -215,73 +234,37 @@ function CreateProject() {
     projectName !== '' &&
     (!shouldCreateCustomRule || conditions?.every?.(condition => condition.value));
 
-  const createProjectForm = (
```

Comment:
I moved this bew because it was just easier for me to debug and find myself in the code, but please let me know if you are ok with that, or of it makes things difficult to review.